src/mcp-bundle/src/Command/McpCommand.php

```diff
@@ -30,7 +30,7 @@ public function __construct(
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $this->server->connect(
-            new SymfonyConsoleTransport($input, $output)
+            new StdioTransport(STDIN, STDOUT)
```

Comment:
needs to be checked, I propose to move the SymfonyConsoleTransport from the MCP SDK to our bundle and use it. Is this a solution @chr-hertel or is the change as is, fine?